RELEASE_NOTES.md

```diff
@@ -1,9 +1,45 @@
+# Release Notes
+
+## stable-1.15-2.0.0, stable-1.16-2.0.0, stable-1.17-2.0.0
+
+* \[awsebscsiprovisioner\] The manual steps to upgrade the snapshot APIs from v1alpha1 to v1beta1 is no longer required. It has been automated in the chart CRD install hook by default. If you do not want that default behavior of cleaning up v1alpha1 snapshot CRDs, you can set `cleanupVolumeSnapshotCRDV1alpha1` to `false` and follow the instructions for upgrading to Kubernetes `1.17`. ([#273](https://github.com/mesosphere/kubernetes-base-addons/pull/273), [@sebbrandt87](https://github.com/sebbrandt87))
+* \[gcpdisk-csi-driver\] The manual steps to upgrade the snapshot APIs from v1alpha1 to v1beta1 is no longer required. It has been automated in the chart CRD install hook by default. If you do not want that default behavior of cleaning up v1alpha1 snapshot CRDs, you can set `cleanupVolumeSnapshotCRDV1alpha1` to `false` and follow the instructions for upgrading to Kubernetes `1.17`.
+  \[azuredisk-csi-driver\] The manual steps to upgrade the snapshot APIs from v1alpha1 to v1beta1 is no longer required. It has been automated in the chart CRD install hook by default. If you do not want that default behavior of cleaning up v1alpha1 snapshot CRDs, you can set `snapshot.cleanupVolumeSnapshotCRDV1alpha1` to `false` and follow the instructions for upgrading to Kubernetes `1.17`. ([#279](https://github.com/mesosphere/kubernetes-base-addons/pull/279), [@jieyu](https://github.com/jieyu))
+* \[prometheus-operator\] Upgrade to version [0.38.1](https://github.com/coreos/prometheus-operator/releases/tag/v0.38.1)
+    - \[prometheus\] Upgrade to version [2.17.2](https://github.com/prometheus/prometheus/releases/tag/v2.17.2)
+    - \[grafana\] Upgrade to version [6.7.3](https://github.com/grafana/grafana/releases/tag/v6.7.3) ([#281](https://github.com/mesosphere/kubernetes-base-addons/pull/281), [@branden](https://github.com/branden))
+* \[traefik\] fix an issue where `clusterhostname` can now be an ipaddress as well ([#286](https://github.com/mesosphere/kubernetes-base-addons/pull/286), [@GoelDeepak](https://github.com/GoelDeepak))
+* [dex-k8s-authenticator] Fix bug in init container that could remove custom CA certificate from main cluster login instructions ([#291](https://github.com/mesosphere/kubernetes-base-addons/pull/291), [@mhrabovcin](https://github.com/mhrabovcin))
+* \[traefik\] Distribute pods across nodes and zones when possible.
```

Comment:
Nice, a lot of improvements in this release!

Can we group all the notes for the same addon together? Seeing a lot of traefik and prometheus notes scattered throughout.

Reply:
I'd prefer not to in order to keep the notes attached to their individual PRs and contributors.

Reply:
> a lot of improvements in this release

Unfortunately, that's because it's been 6 weeks since the last stable release.

Reply:
you'd still be able to maintain that info. it's just bringing them under the same grouping so it's easier to see all the updates for that addon e.g.
```
[traefik]
- fix an issue where `clusterhostname` can now be an ipaddress as well ([#286](https://github.com/mesosphere/kubernetes-base-addons/pull/286), [@GoelDeepak](https://github.com/GoelDeepak))
- access log is enabled by default ([#305](https://github.com/mesosphere/kubernetes-base-addons/pull/305), [@mhrabovcin](https://github.com/mhrabovcin))
- ...
```

Reply:
Let's get the Makefile that does this merged first so the process is all in code, then work on automating that in future iterations?